src/cascadia/TerminalApp/Resources/de-DE/Resources.resw

```diff
@@ -1,939 +1,972 @@
-﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
FWIW you should revert the changes to all the Resources.resw files that aren't in the `en-us` dir. All the other locales are auto-generated from the en-us one by a separate build pipeline

Reply:
you right, reverted it

Reply:
done